.gitpod.Dockerfile

```diff
@@ -0,0 +1,25 @@
+# You can find the new timestamped tags here:
+# https://hub.docker.com/r/gitpod/workspace-python-3.9/tags
+# It's best to pin a version instead of relying on "latest"
+# for full reproducibility.
+FROM gitpod/workspace-python-3.9:2022-09-07-02-19-02
+
+CMD [ "/bin/bash" ]
+
+ARG CONDA_VERSION=py39_4.12.0
+ARG MINICONDA_URL="https://repo.anaconda.com/miniconda/Miniconda3-${CONDA_VERSION}-Linux-x86_64.sh"
+
+# Install conda and mamba, based on instructions here:
+# https://github.com/ContinuumIO/docker-images/tree/master/miniconda3/debian
+RUN wget "${MINICONDA_URL}" -O ~/miniconda.sh -q && \
+    sudo sh ~/miniconda.sh -b -p /opt/conda && \
+    rm ~/miniconda.sh && \
+    sudo ln -s /opt/conda/etc/profile.d/conda.sh /etc/profile.d/conda.sh && \
+    echo ". /opt/conda/etc/profile.d/conda.sh" >> ~/.bashrc && \
+    sudo /opt/conda/bin/conda install mamba=0.25.0 -c conda-forge -y && \
+    sudo find /opt/conda/ -follow -type f -name '*.a' -delete && \
+    sudo find /opt/conda/ -follow -type f -name '*.js.map' -delete && \
+    sudo /opt/conda/bin/conda clean -afy
+
+# This is Gitpod only. The production version on AWS should comment this out.
```

Comment:
I don't think I understand this comment.

Reply:
In the context of this repository, it actually is meaningless, so I am happy to remove! But for a bioinformatics workflow, you can image doing your development on Gitpod and actually running your full workflow on AWS. I'd like to create the AWS environment from the Dockerfile for full reproducibility. 

The following line transfers the ownership of the contents of the container to the gitpod user so you can install new conda packages etc. This is because you don't have root access on Gitpod. That would not be the case on AWS. 

So you technically wouldn't need this line on AWS, though perhaps a cleaner way to do that would be to have a development Dockerfile (used on Gitpod) and a production Dockerfile (used on AWS) which is a pretty standard practice.